app/models/sign_in/access_token.rb

```diff
@@ -61,6 +67,9 @@ def initialize(session_handle:,
       @version = version || Constants::AccessToken::CURRENT_VERSION
       @expiration_time = expiration_time || set_expiration_time
       @created_time = created_time || set_created_time
+      @first_name = first_name
```

Comment:
I think having separate params for each of these will get out of hand real fast if we ever add more user attributes. What we prob should do instead is take a param that's a `user_attributes` hash, and validate here that the keys are included in the values configured in the `ClientConfig`

Reply:
actually we can just filter for the values configured in the `ClientConfig`, that way entities that create the `AccessToken` can just pass the entire `user_attributes` hash and let `AccessToken` figure out which fields to pick from